Correct overlooked bigendian case.  This has been fixed upstream in a different
way since the 1.0.30 release.

--- a/backend/genesys/low.cpp
+++ b/backend/genesys/low.cpp
@@ -539,7 +539,7 @@
     }
 
 #ifdef WORDS_BIGENDIAN
-    if (depth == 16) {
+    if (session.params.depth == 16) {
         dev->pipeline.push_node<ImagePipelineNodeSwap16BitEndian>();
     }
 #endif
